mappings/net/minecraft/util/ToStringTransformer.mapping

```diff
@@ -0,0 +1,23 @@
+CLASS net/minecraft/class_6384 net/minecraft/util/ToStringTransformer
```

Comment:
Imo this should go to `gui/screen/narration` as this is used to generate sentences for any arbitrary input. When I mapped this, I called this `Narration` for this contains the data for narration in `value`; and I named the `transformer` a `feeder` as it feeds sentences into the given consumer (seen as a reader)